app/controllers/users/edit_email_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class EditEmailController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
+    def edit
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
+    end
+
+    def update
+      @update_user_email_form = UpdateUserEmailForm.new(current_user)
```

Comment:
Also, when reading a particular view file, or jumping back and forth between various similar views, it might not be immediately clear which view you're looking at, so having a well-named variable goes a long way I think.